packages/wallet/test/mocks/mockUtxoProvider.ts

```diff
@@ -0,0 +1,13 @@
+import { UtxoProvider } from '@cardano-sdk/core';
+
+/**
+ * Provider stub for testing
+ *
+ * returns UtxoProvider-compatible object
+ */
+export const mockUtxoProvider = (): UtxoProvider => ({
+  healthCheck: jest.fn().mockResolvedValue(true),
+  utxoByAddresses: jest.fn().mockResolvedValue([])
```

Comment:
Should move the resolved utxo from original mock wallet provider. Tests rely on it